grid2op/Action/SerializableActionSpace.py

```diff
@@ -76,21 +76,74 @@ def sample(self):
         """
         A utility used to sample a new random :class:`Action`.
 
```

Comment:
a bit of documentation maybe, to emphasize that "sample" will sample uniformely on the unary actions and not among all possible actions.